### DISCUSSION TITLE
review commit github- ignore whitespaces

### DISCUSSION THREAD

**Discussion by yeonatanPerelman** [FPGA-MAFIA/fpga_mafia#35] [Q&A] [answered]
when you want to review commits changes in github, you want to ignore white spaces. add to the end of URL: ?w=1  
example:  
> `https://github.com/amichai-bd/riscv_manycore_mesh_fpga/commit/090ab95699358d9729a7343f3e59ef412e252a81`  

> `https://github.com/amichai-bd/riscv_manycore_mesh_fpga/commit/090ab95699358d9729a7343f3e59ef412e252a81?w=1`  

## with white spaces
![image](https://user-images.githubusercontent.com/111231093/203139981-bf20d1db-22c9-467e-92b1-35456108a7ed.png)

## ignore white spaces
![image](https://user-images.githubusercontent.com/111231093/203140095-62e0346b-4a82-4106-8de7-7b020c847741.png)

**Comment by amichai-bd** [comment, marked as answer]
Marking this as answered and locking conversation
